SocketUtils.cpp

```diff
@@ -44,6 +44,20 @@ bool SocketUtils::getNameInfo(const struct sockaddr *sa, socklen_t salen,
   return true;
 }
 
+int SocketUtils::getaddrfamily(const char *addr) {
+  struct addrinfo hint, *info = 0;
+  memset(&hint, 0, sizeof(hint));
+  hint.ai_family = AF_UNSPEC;
+  // Uncomment this to disable DNS lookup
+  // hint.ai_flags = AI_NUMERICHOST;
+  int ret = getaddrinfo(addr, 0, &hint, &info);
+  if (ret)
```

Comment:
missing {}
also probably should log the error instead of just silent -1

I think we have proper usage of getaddrinfo elsewhere in this file